src/test/kotlin/no/nav/melosys/skjema/service/InnsendingStatusServiceTest.kt

```diff
@@ -0,0 +1,189 @@
+package no.nav.melosys.skjema.service
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.mockk.clearMocks
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import no.nav.melosys.skjema.domain.InnsendingStatus
+import no.nav.melosys.skjema.entity.Innsending
+import no.nav.melosys.skjema.entity.Skjema
+import no.nav.melosys.skjema.entity.SkjemaStatus
+import no.nav.melosys.skjema.repository.InnsendingRepository
+import no.nav.melosys.skjema.repository.SkjemaRepository
+import java.util.*
+
+class InnsendingStatusServiceTest : FunSpec({
```

Comment:
Jeg tenker at man like gjerne kan bruke ekte repository-instanser i denne testen istedenfor mocking mtp at det populere de med data for testene i min erfaring er mindre knotete enn å mocke ut oppførselen. Da kan man gjøre assertions på state i databasen istedenfor captured metode-kall. Da blir testene også mer robuste